ivy/functional/frontends/paddle/vision/transforms.py

```diff
@@ -135,3 +135,10 @@ def vflip(img, data_format="CHW"):
     elif data_format.lower() == "hwc":
         axis = -3
     return ivy.flip(img, axis=axis)
+
```

Comment:
Please revert the changes in the `.idea` folder. These shouldn't get merged with your PR.